src/server_manager/web_app/main.ts

```diff
@@ -41,10 +37,7 @@ document.addEventListener('WebComponentsReady', () => {
   const metricsUrl = ensureString(queryParams.metricsUrl);
   const shadowboxImage = ensureString(queryParams.image);
   const version = ensureString(queryParams.version);
-  const sentryDsn = ensureString(queryParams.sentryDsn) || DEFAULT_SENTRY_DSN;
-
-  // Initialize error reporting.
-  SentryErrorReporter.init(sentryDsn, version);
+  const sentryDsn = ensureString(queryParams.sentryDsn);
```

Comment:
Shouldn't you keep the default logic here? Otherwise you would need it in different places.

Reply:
No, the main process always sends over the right DSN. I don't want to duplicate the default DSN is two places.